pkg/sampler/sampler.go

```diff
@@ -57,65 +57,56 @@ func (e StatusCodeError) Error() string {
 	)
 }
 
-// Sampler implements Sampler, using http.Transport.
-type Sampler struct {
-	tr        http.Transport
-	UserAgent string
-}
+// Ping measures a given URL and returns a Sample
+func Ping(target Target, timeout int) (sample Sample, err error) {
```

Comment:
How strongly do you feel about the name `Ping`? Why not something like `Measure`/`Sample`/`Check`? while I know what its doing, I feel the term `Ping` has strong connections to ICMP for various folk.